[Doc] [runtime env] Clarify zip requirements for remote URIs (#28026)
ray-project/ray: doc/source/ray-core/handling-dependencies.rst

  You can inspect a zip file's contents by running the ``zipinfo -1 zip_file_name.zip`` command in the Terminal.
  Some zipping methods can cause hidden files or metadata directories to appear in the zip file at the top level.
  This will cause Ray to throw an error because the structure of the zip file is invalid since there is more than a single directory at the top level.

>   You can avoid this by using the ``zip -r`` command directly on the directory you want to compress and being sure to run the command from the parent directory.

Reviewer: ```suggestion
  You can avoid this by using the ``zip -r`` command directly on the directory you want to compress. Make sure to run the command from that directory's parent.
```

Author: Thanks, this is better
